easybuild/easyconfigs/t/triqs_cthyb/triqs_cthyb-2.2.0-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,70 @@
+easyblock = 'CMakeMake'
+
+name = 'triqs_cthyb'
+version = '2.2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://triqs.github.io/cthyb/'
+description = """
+ TRIQS (Toolbox for Research on Interacting Quantum Systems) is a
+ scientific project providing a set of C++ and Python libraries to
+ develop new tools for the study of interacting quantum systems.
+
+ cthyb = continuous-time hybridisation-expansion quantum Monte Carlo
+
+ The TRIQS-based hybridization-expansion solver allows to solve the
+ generic problem of a quantum impurity embedded in a conduction bath
+ for an arbitrary local interaction vertex. The “impurity” can be any
+ set of orbitals, on one or several atoms.
+"""
+
+docurls = ['https://triqs.github.io/cthyb/%(version_major_minor)s.x/']
+software_license = 'LicenseGPLv3'
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://github.com/TRIQS/cthyb/releases/download/%(version)s/']
+sources = ['cthyb-%(version)s.tar.gz']
+checksums = ['89b572027f88c6b9afd176bd36023d646efba056ce90dea1af57b662fd713ad5']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('SciPy-bundle', '2019.03'),
+    ('Boost', '1.70.0'),
+    ('Clang', '8.0.0'),
+    ('GMP', '6.1.2'),
+    ('HDF5', '1.10.5'),
+    ('Mako', '1.0.8'),
+    ('h5py', '2.9.0'),
+    ('triqs', '2.2.1', versionsuffix),
```

Comment:
Is there a way to loosen this to be compatible against any version `2.2.x` ?
We develop triqs and the applications with this compatibility in mind.

Reply:
The dependency is to a specific version.

Reply:
Indeed, currently all dependency versions are hard fixed in easyconfigs.

There are some ideas about slightly loosening that up, but it's a bit of a can of worms (has all sorts of implications), and nobody is really looking into that currently afaik.